server/tilesource/base.py

```diff
@@ -200,12 +457,16 @@ def canRead(cls, *args, **kwargs):
         return False
 
     def getMetadata(self):
+        mag = self.getMagnification()
         return {
             'levels': self.levels,
             'sizeX': self.sizeX,
             'sizeY': self.sizeY,
             'tileWidth': self.tileWidth,
             'tileHeight': self.tileHeight,
+            'magnification': mag['magnification'],
+            'pixelWidthInMillimeters': mag['mm_x'],
```

Comment:
Maybe just use `mm_x` and `mm_y`. That way people there is only way to access these properties in the outputs of `getMetadata()` and `getMagnification()`.